### PR TITLE
TF Keras tf_utils_test missing test cases add

### PR DESCRIPTION
**Pull request by Dayananda-V**
1-Certain basic unit test cases add in tf_utils_test(//tensorflow/python/keras:tf_utils_test)
2-testGetReachableFromInputs test case move under right package

### CODE REVIEW

**Comment by Dayananda-V** [comment]
@jvdillon 

Welcome for review and thanks.....

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @jvdillon: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @jvdillon: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 29 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @jvdillon: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 44 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @jvdillon: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 59 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @jvdillon: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 76 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @pavithrasv: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @pavithrasv: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 29 days with no activity and the `awaiting review` label has been applied.

File: tensorflow/python/keras/utils/tf_utils_test.py [hunk 1]
```diff
@@ -20,14 +20,82 @@
...
 from tensorflow.python.framework import sparse_tensor
 from tensorflow.python.framework import test_util
 from tensorflow.python.keras.utils import tf_utils
+from tensorflow.python.ops import array_ops
 from tensorflow.python.ops import variables
 from tensorflow.python.platform import test
 
 
+class ConstantValueTest(test.TestCase):
+
```
Comment (pavithrasv) [inline tensorflow/python/keras/utils/tf_utils_test.py]: I think we need to test Variable case as that is the only additional use case tf_utils.smart_cond is solving. Otherwise we just redirect to the underlying smart_cond API.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by tensorflowbutler** [comment]
It has been 43 days that this pull-request has stalled. Please create a new pull-request with the requested changes.
